sycl/test/check_device_code/vector/vector_as.cpp

```diff
@@ -0,0 +1,48 @@
+// NOTE: Assertions have been autogenerated by utils/update_cc_test_checks.py UTC_ARGS: --functions "as" --include-generated-funcs --version 4
```

Comment:
clang-format complains here but that line has to stay as the script needs it like this. Normally it's used under `clang/test` and long lines are allowed there.